libraries/rush-lib/src/logic/operations/test/AsyncOperationQueue.test.ts

```diff
@@ -60,15 +67,20 @@ describe(AsyncOperationQueue.name, () => {
     const queue: AsyncOperationQueue = new AsyncOperationQueue(operations, customSort);
     for await (const operation of queue) {
       actualOrder.push(operation);
+      if (operation === UNASSIGNED_OPERATION) {
+        continue;
+      }
```

Comment:
Likewise, this should throw if it occurs, since nothing is entering `RemoteExecuting` status and that's the only path that should be returning `UNASSIGNED_OPERATION`.